modules/ROOT/pages/anypoint-platform-cli.adoc

```diff
@@ -78,7 +78,7 @@ If you send both username and password, and the token bearer, the CLI prioritize
 
 == Usage
 
-=== Loggin In
+=== Logging In
```

Comment:
It's incorrect, it should be 

> Log in

Reply:
I'd say "Logging In" sounds better as it describes the process. While "Log In" sounds uncertain.